compiler/modules/comm.diagnostic/diagnostic.bal

```diff
@@ -25,6 +25,7 @@ public type File readonly & object {
     public function filename() returns string;
     public function directory() returns string?;
     public function lineColumn(Position pos) returns LineColumn;
+    public function lineContent(Range|Position range) returns string[];
```

Comment:
The parameter should be a `Position` and the return value should be a `string`.

Reply:
fixed